### ISSUE TITLE
Remind readers that what you see/don't see doesn't always reflect reality

### ISSUE THREAD

**Issue by luyangkenneth** [sushinoya/singapore-tech-internships#10]
Maybe we should remove the "Application Period" column and put a note that says that companies:
- might offer internships even if they're not listed
- might offer internships all-year-round even if they don't publicize it
- might offer internships for more kinds of roles than just the ones listed

...and that it's really up to you as an individual to seek out available opportunities.

As for the cases that are pretty set in stone - e.g. Google and Facebook SG only hosting a summer internship (for now) - maybe we can just indicate in the remarks?

**Comment by luyangkenneth** [comment]
It's not obvious to me how I would best phrase all this - so I'm leaving this up for grabs :)

**Comment by raivatshah** [comment]
I agree to this! More clarify would definitely help!

**Comment by sushinoya** [comment]
I know that I'm coming back to it after ages but I feel like there is some merit to keeping the "Application Period" column because it clearly lets people see which companies they should reach out to first when reaching out for an off-cycle internship. We could def add that in the remarks too but i feel like having a column makes it easier to scan. I think updating the description is useful. Thoughts @luyangkenneth @raivatshah @aaditkamat?

**Comment by luyangkenneth** [comment]
> it clearly lets people see which companies they should reach out to first when reaching out for an off-cycle internship

Yeah keeping the Application Period column makes sense to me! I think the spirit of this topic is about setting the right mindset when using this list of companies.
